afs/config_handler.py

```diff
@@ -131,6 +133,8 @@ def next_node(self, data=None, debug=False):
                 return self.flow_obj.exe_next_node(data, debug=debug)
             else:
                 raise AssertionError('Type error, data must be DataFrame type')
+        else:
+            raise AssertionError('Data cannot be none')
```

Comment:
I would prefer to check type only, cause `None` is also a `TypeError`, not sure `is not None` still necessary or not. And, if this function is not allow data is `None`, why you use it as the default value? BTW, in your error message, you said it's `Type error`, but here is raising `AssertionError`, does this error clear enough?

Reply:
1. Because `data` may be one of the send type, I give the default value. However, there is unknown requirement in the future. I would fix it.
2. Error message should be shown as the respond to correct type. I would fix it.